plumbing/protocol/packp/capability/list.go

```diff
@@ -0,0 +1,158 @@
+package capability
```

Comment:
This is not in the scope of this object this should be checked in the request since are request dependant